src/pages/home/Home.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import 'i18n/i18n';
+import { translate } from 'react-i18next';
+import styles from './home.scss';
+
+const logo = require('assets/logo.png');
+
+const Home = ({ t }) => {
+  return (
+    <div className={styles.homeContainer}>
+      <div className={styles.mainHeaderBackground} />
+      <nav className={styles.navigationBar}>
```

Comment:
It would be nice if we can create a reusable component for the main navigation, later on we will need to incorporate in other pages.

Reply:
Yes, re-usable is key.